docs/howto/manage-charms.rst

```diff
@@ -250,6 +255,26 @@ that at present this must be a Discourse page. E.g.,
 
     See more: :ref:`charmcraft-yaml-key-links`
 
+Smaller charms can have a single-page documentation in the Description tab. 
+Bigger charms with multi-page documentation, should have a full
+[Diátaxis](https://diataxis.fr/) navigation tree in the Description tab, 
+or just provide a brief description with a link to an external documentation set. 
+
+.. admonition:: Best practice
+    :class: hint
+
+    Smaller charm documentation exemplars:
```

Comment:
> exemplar

This term came up before in a discussion about user-facing docs. Do you think for ease of understanding we should use "example" instead?

Reply:
Yes, I see how the word example can be easier to understand. Although the meaning is not precisely the same, I guess in this context we can replace it. Fixed in https://github.com/canonical/charmcraft/pull/2253/commits/8ef59e0da29fd0535a9d45fe03fa8ec90af3ef13